server/routes/routes.js

```diff
@@ -36,4 +37,19 @@ router.get('/', (req, res) => {
 	});
 });
 
+router.route('/sighting')
```

Comment:
routes should only be for navigation routes.

api endpoints should be in their own `api` file, and routed to an api route to differentiate/version

Reply:
This should be complete now.